### PR TITLE
Fix build on windows host

### PR DESCRIPTION
**Pull request by aleksuss**
Fixes issue #411

### CODE REVIEW

File: librocksdb-sys/build.rs [hunk 1]
```diff
@@ -145,7 +145,8 @@ fn build_rocksdb() {
             .iter()
             .cloned()
             .filter(|file| match *file {
-                "port/port_posix.cc" | "env/env_posix.cc" | "env/io_posix.cc" => false,
+                "port/port_posix.cc" | "env/env_posix.cc" | "env/fs_posix.cc"
+                | "env/io_posix.cc" => false,
```
Comment (ordian) [inline librocksdb-sys/build.rs:149]: can it just be `!file.ends_with("posix.cc")`?

Comment (aleksuss) [reply librocksdb-sys/build.rs:149]: Actually can. But I prefer such approach. What if in the future will appear file with name which ends with `posix.cc` but relates to both OS ? We'll skip it.

Comment (ordian) [reply librocksdb-sys/build.rs:149]: Right, but if a file name ends with posix.cc, it's almost certainly POSIX-specific. OTOH, it's easy to miss one such file when updating the sources as we can see here.

Comment (ordian) [reply librocksdb-sys/build.rs:149]: It doesn't matter in the end.
The real problem is a lack of (required) windows CI check.

Comment (aleksuss) [reply librocksdb-sys/build.rs:149]: @ordian exactly. By the way I'm trying to fix travis now 🙂

**Comment by ordian** [comment]
Can confirm the fix https://ci.appveyor.com/project/paritytech/parity-common/builds/32383803#L508.

**Pull request merged**